icu4c/APIChangeReport.md

```diff
@@ -5,233 +5,460 @@
  License & terms of use: http://www.unicode.org/copyright.html
 -->
 
-# ICU4C API Comparison: ICU 71 with ICU 72
+# ICU4C API Comparison: ICU 72 with ICU 73
 
 > _Note_ Markdown format of this document is new for ICU 65.
 
-- [Removed from ICU 71](#removed)
-- [Deprecated or Obsoleted in ICU 72](#deprecated)
-- [Changed in  ICU 72](#changed)
-- [Promoted to stable in ICU 72](#promoted)
-- [Added in ICU 72](#added)
-- [Other existing drafts in ICU 72](#other)
+- [Removed from ICU 72](#removed)
```

Comment:
We can remove stuff that's marked `@stable`?

Reply:
Actually, on closer examination, it looks like all of these entries show up again below in the "additions" chart.  I think the removal of the `U_OVERRIDE` keyword caused whatever tool is generating this report to not see them as the same function.

Reply:
Thanks for the review. Hmmm, another example of tooling needing updates. Will work on this later today.

Reply:
I think this is due to this change: https://github.com/unicode-org/icu/commit/a3cbe809091807b406cf526635cb866270a20da2#diff-35fa94c5c8ddb5fa01c3cfb8a0fb1c7f36d6500ad48b0afc0df97660f0c68de4

In StableAPI.java, please change `"[ ]*override", "", // remove override` back to `"[ ]*(override|U_OVERRIDE)", "", // remove U_OVERRIDE and override`, and then regenerate. (Make sure to copy-paste this exactly.)

Reply:
Update done to StableAPI.java, fixing the reports.